package.json

```
@@ -5,10 +5,12 @@
   "packageManager": "npm@11.5.2",
   "workspaces": [
     "src/examples/standalone/vercel",
+    "src/examples/standalone/langchain",
     "src/examples/telegram/vercel"
   ],
   "scripts": {
     "start:standalone-vercel": "turbo run interact --filter=./src/examples/standalone/vercel --ui=tui",
+    "start:standalone-langchain": "turbo run interact --filter=./src/examples/standalone/langchain --ui=tui",
```

```suggestion
    "start:standalone-langchain": "npm start --prefix ./src/examples/standalone/langchain",
```

I did this with the other ones, dont change the other ones so you get fewer conflicts